### ISSUE TITLE
[Lab3-2] Question about loss function of discriminator

### ISSUE THREAD

**Issue by doom8199** [2017-fall-DL-training-program/VAE-GAN-and-VAE-GAN#7]
Dear TA:
    I tried to modify sample code below, but finally got the garbage results QQ ...
    Could you give me some hints why this change cannot work ?
================
errD_real = criterion(...)
#errD_real.backward()

errD_fake = criterion(...)
#errD_fake.backward()

errD = errD_real + errD_fake
errD.backward()
===================

[Note] Training dataloader has been followed by Lab3_2's requirement.

**Comment by hui-po-wang** [comment]
Hi @doom8199,

At first glance, they should be equal if you didn't modify anything else.
It's hard to debug without any details.

Thanks

**Comment by doom8199** [comment]
Dear TA:
    Details are just code changes I mentioned before. I will try to run more than one time to see if it can be reproducible or not.
